index.html

```diff
@@ -27,7 +27,8 @@
         div{margin:15px 0px;}
         .error{color:#ff1111;}
         .field{width:100%;max-width:600px;}
-        body{background-color:#fafafa;}
+		h1,h2,h3 { line-height: 1.2 }
```

Comment:
this line uses tabs, unlike the rest of the file which uses spaces to indent. (just a note, i'm not associated with the project in any way, I just know this can be annoying for some people)